Pyjion/pyjit.cpp

```diff
@@ -517,53 +564,136 @@ void PyjionJitFree(PyjionJittedCode* function) {
         delete code;
         g_pyjionJittedCode.erase(function);
     }
-#else
-    delete (TraceInfo*)function->j_evalstate;
 #endif
+	delete obj;
+}
+
+static PyObject *pyjion_enable(PyObject *self, PyObject* args) {
+	auto ts = PyThreadState_Get();
+	auto prev = ts->interp->eval_frame;
+	ts->interp->eval_frame = PyJit_EvalFrame;
+	return prev == PyJit_EvalFrame ? Py_False : Py_True;
+}
+
+static PyObject *pyjion_disable(PyObject *self, PyObject* args) {
+	auto ts = PyThreadState_Get();
+	auto prev = ts->interp->eval_frame;
+	ts->interp->eval_frame = _PyEval_EvalFrameDefault;
```

Comment:
I could see this causing issues if you are debugging in e.g. PyCharm. However running two custom EvalFrame functions at the same time might be... tenuous. Im just thinking it would be bad to lose the debugger part way through.